driver.sh

```diff
@@ -142,6 +142,7 @@ check_dependencies() {
   # Check for existence of needed binaries
   command -v nproc
   command -v "${CROSS_COMPILE:-}"as
+  command -v readelf
```

Comment:
why you no use `llvm-readelf`??? :see_no_evil: :monkey: :monkey_face:

Reply:
TIL that was a thing :P

We would need check for the different available versions like `clang` and `llvm-ar` though but that'd not hard.